### PR TITLE
add r-pcamixdata

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

File: recipes/r-pcamixdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('PCAmixdata')"           # [not win]
+    - "\"%R%\" -e \"library('PCAmixdata')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=PCAmixdata
+  license: GPL (>= 2.0)
```
Comment (dbast) [inline recipes/r-pcamixdata/meta.yaml:38]:
```suggestion
  license: GPL-2.0-or-alter
```

File: recipes/r-pcamixdata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  summary: Implements principal component analysis, orthogonal rotation and multiple factor analysis
+    for a mixture of quantitative and qualitative variables.
+  license_family: GPL3
+  license_file:     
+    - LICENSE
```
Comment (dbast) [inline recipes/r-pcamixdata/meta.yaml:43]:
```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
    - LICENSE
```

File: recipes/r-pcamixdata/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+extra:
+  recipe-maintainers:
+    - FernandezMathieu
```
Comment (dbast) [inline recipes/r-pcamixdata/meta.yaml:48]:
```suggestion
    - conda-forge/r
    - FernandezMathieu
```

**Pull request merged**